src/bootstrap/src/core/config/config.rs

```diff
@@ -2756,9 +2756,19 @@ impl Config {
 
         // If `download-rustc` is not set, default to rebuilding.
```

Comment:
Question: this comment is somewhat misleading in that it's more like "if `download-rustc` is not set in config.toml or via cli flags" right?